### ISSUE TITLE
CMIP7 MIP Table Identifiers

### ISSUE THREAD

**Issue by martinjuckes** [CMIP-Data-Request/Harmonised-Consultation-Phase-1#26]
The CMIP6 Plus approach gives a lot of improvements over CMIP6, but would also create a disruption in the naming of the most widely used CMIP variables which are generally in tables which have had stable names for multiple CMIP eras.  

The CMIP6 Plus table list could be modified to reduce the disruption caused by unneeded changes to long-standing terms. This would involve replacing “AP...” with “A”, “OP...” with “O”, and removing the realm prefix from sub-daily and fixed fields. The “GI....” and “AI....” tables are also redundant in the CMIP AR7 Fast track request as the region specification has been moved from being implicit in the table name to being a separate element in a the variable group specification. Retain “CF..” tables which are widely used and omit the “AC..” tables which have minimal content, and the “AE..” tables which have taken some of the “CF...” content.

·	CFday, CFmon, CFmonLev, CFmonZ, CFsubhrPt, CFsubhrPtSite, 
·	1hr, 1hrPt, 3hr, 3hrPt, 3hrPtLev, 6hr, 6hrPt, 6hrPtLev, 6hrPtZ, fx
·	Aday, AdayLev, AdayZ, Amon, AmonClim, AmonClimLev, AmonDiurnal, AmonLev, AmonZ, AsubhrPt,  
·	LIday, LImon, 
·	Lday, Lmon, Lyr, LyrPt, 
·	OBday, OBmon, OBmonLev, OByr, OByrLev, 
·	Oday, Odec, OdecLev, OdecZ, Omon, OmonClim, OmonClimLev, OmonLev, OmonZ, Oyr, OyrLev, 
·	SIday, SImon, SImonPt.

This still results in an increase in the number of MIP tables (51) compared to CMIP6, but considerably less than in CMIP6 Plus, while still retaining a more systematic approach than CMIP6. It also retains continuity with CMIP6 and earlier MCIP phases for the most widely used variables.

**Comment by durack1** [comment]
Just tagging that the https://github.com/PCMDI/cmip6-cmor-tables/ and the https://github.com/PCMDI/mip-cmor-tables/ are relevant references for the discussion above

**Comment by durack1** [comment]
It's useful to put the logic in the same place so we can have a meaningful discussion.

Just for context, this is what the evolution looks like, total number of tables per mip_era, [cmip3-cmor-tables](https://github.com/PCMDI/cmip3-cmor-tables/tree/master/Tables) = 6, [cmip5-cmor-tables](https://github.com/PCMDI/cmip5-cmor-tables/tree/master/Tables) = 18, [cmip6-cmor-tables](https://github.com/PCMDI/cmip6-cmor-tables/tree/main/Tables) = 44 and [CMIP6Plus/mip-cmor-tables](https://github.com/PCMDI/mip-cmor-tables/tree/main/Tables) = 81, remapping all entries (including CMIP6 E* tables) into 9 identifiers AC = Atmosphere chemistry, AE = Atmosphere Aerosol, AP = Atmosphere Physics, GI = Greenland Icesheet, LI = Land Ice, LP = Land Physics, OB = Ocean Biogeochemistry, OP = Ocean Physics, SI = Sea Ice).

For historical context, the table evolution had originally been targeting the model components (or history files) that the data would be written from, this evolved a lot in CMIP5 and subsequent phases, with the CF tables being merged into CMIP5 (after having been generated in parallel to the CMIP3 tables, the [cfmip1-cmor-tables](https://github.com/PCMDI/cfmip1-cmor-tables)).

To further these discussions I wonder whether knowing how many variables, and duplicates exist across these tables could be a good idea? In the variable registry discussions that we've been having for months, this was the intended next steps, which I do believe @wolfiex had started - not sure where we're up to.

ping @matthew-mizielinski @taylor13

**Comment by taylor13** [comment]
regarding duplication of variables:  Of the 2062 CMIP6 variables, about 25% differ only by the sampling interval ("frequency") or the region (Greenland, Antarctica, global) they were requested at.  All the variable attributes and all the CF-convention global attributes are the same across duplicate variables.

**Comment by martinjuckes** [comment]
@taylor13 , @durack1 :

Some background on the list of identifiers that I've suggested above:
  * 6 table identifiers for regions (Iyr,Omon,Ifx/Ant,Gre) retired: these were needed in CMIP6 but the region is now specified in the variable group and the relevant variables can be assigned the LImon etc table identifiers.
  * Retire all the `E....` table identifiers. These were introduced for technical reasons in CMIP6 and are not needed. Variables can use existing identifiers such as `Amon` etc.
  * Retain well established identifiers such as `Amon`, `Omon`, `CFmon` and many others. These well established names help a broad user base with to engage with the request without navigating technical details.
  * Retire `Ofx`, `Efx`, `AERfx`: these are all about the model configuration and there is no need for separate identifiers.
  * Rationalise use of modifiers `Clim`, `Pt`, etc.
  
  The main differences from the larger list proposed in CMIP6Plus are:
  * Retaining long-used names such as `Amon`, `Omon` to avoid making changes to well used variables. 
  * Less systematic, retaining an element of putting things together in a way which is convenient for the community rather than through a careful algorithm.

**Comment by martinjuckes** [comment]
Data Request WIP meeting on Oct 1st 2024 concluded with a decision to stay with the CMIP6 table names. 

Meeting notes (restricted access): [here](https://office.wcrp-cmip.org/Products/Files/DocEditor.aspx?fileid=10005)
